GCRCatalogs/dc2_truth.py

```diff
@@ -11,36 +12,46 @@ class DC2TruthCatalogReader(BaseGenericCatalog):
     native_filter_string_only = True
 
     def _subclass_init(self, **kwargs):
+        self._filename = kwargs['filename']
 
-        if not os.path.isfile(kwargs['filename']):
-            raise ValueError("%s is not a valid filename" % kwargs['filename'])
+        if not os.path.isfile(self._filename):
+            raise ValueError('{} is not a valid file'.format(self._filename))
```

Comment:
I'm just curious: is there a reason you chose to change this line?  The new line and the old one look effectively identical to me, unless there is some reason to prefer `format` that I am not aware of...

Reply:
Nope, only the consistency with other code in this package. If there's a strong reason to use `%` we can revert it back.